kernel/src/platform/snp.rs

```diff
@@ -345,6 +345,14 @@ impl SvsmPlatform for SnpPlatform {
     fn start_svsm_request_loop(&self) -> bool {
         true
     }
+
+    fn mmio_write(&self, paddr: PhysAddr, data: &[u8]) -> Result<(), SvsmError> {
```

Comment:
I tried `fn mmio_write<T: IntoBytes>(&self, paddr: PhysAddr, data: &T) -> Result<(), SvsmError> ` but SVSM_PLATFORM can't have a type parameter for that function